consolida.py

```diff
@@ -138,13 +138,18 @@ def parse_state_file(self, response):
                     "deaths": deaths,
                 }
                 result.append(row)
+
+        result = sorted(result, key=lambda row:row["date"])
         row_key = lambda row: (row["state"], row["city"], row["place_type"])
         groups = groupby(result, key=row_key)
-        is_last = {}
+        count_groups = {}
         for key, row_list in groups:
-            is_last[key] = max(row["date"] for row in row_list)
+            for row in row_list:
+                count_groups[key] = count_groups.get(key, 0)+1
```

Comment:
Poderia rodar `black` nesse arquivo depois de suas alterações? Ele implementa a PEP8.

Reply:
Black for executado!